releasenotes/notes/set-downloader-exec-dir-152d9f3d62355ef7.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
```

Comment:
we don't need a releasenote on a fix for a new feature as this was never customer facing.

Reply:
Should I delete the reno slug then?

Reply:
yes please do, and add the `noreno` label to your PR